beanprice/price.py

```diff
@@ -638,8 +651,8 @@ def filter_redundant_prices(price_entries, existing_entries, diffs=False):
     existing_prices = {(entry.date, entry.currency): entry
                        for entry in existing_entries
                        if isinstance(entry, data.Price)}
-    filtered_prices = []
-    ignored_prices = []
+    filtered_prices: List[data.Price] = []
+    ignored_prices: List[data.Price] = []
```

Comment:
Again, I don't think typing variables within function definitions is that useful.

Reply:
When I remove the type information, `mypy` will fail with:

```
beanprice/price.py:657: error: Need type annotation for 'filtered_prices' (hint: "filtered_prices: List[<type>] = ...")
```